packages/accounts/scripts/plugingen.ts

```diff
@@ -78,20 +78,38 @@ export function plugingen({
         const executionAbiConst = `${contract.name}ExecutionFunctionAbi`;
 
         const encodeFunctions = executionAbi.map((n) => {
+          const methodContent = [];
           const argsParamString =
             n.inputs.length > 0
               ? `{ args }: GetFunctionArgs<typeof ${executionAbiConst}, "${n.name}">`
               : "";
           const argsEncodeString = n.inputs.length > 0 ? "args," : "";
-          return dedent`
-                encode${pascalCase(n.name)}Data: (${argsParamString}) => {
-                    return encodeFunctionData({
-                        abi: ${executionAbiConst},
-                        functionName: "${n.name}",
-                        ${argsEncodeString}
-                    })
-                }
-            `;
+          const isViewFunction = n.stateMutability === "view";
```

Comment:
for non-view functions, we will want to include them as part of an object called `providerDecorators`